subscriptions/management/commands/add_prepend_next_to_subscriptions.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("Active subscription holders need to be informed via audio file "
```

Comment:
I would change this to something more generic, we might use this command in future to inform users of other new functionality.